ui/src/app/pages/facility/component/providers/providers-add/providers-add.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit } from '@angular/core';
+import { ProvidersModule } from '../providers.module';
+import {Router,ActivatedRoute} from '@angular/router';
+import { DcimService } from '../../dcim/dcim.service';
+@Component({
+  selector: 'app-providers-add',
+  templateUrl: './providers-add.component.html',
+  styleUrls: ['./providers-add.component.scss']
+})
+export class ProvidersAddComponent implements OnInit {
+  id:any;
```

Comment:
Because we already define these properties in Module.We can use object of the module when we need a two-way bingding.